Sources/EpoxyBars/BarInstaller/SelectableStackView.swift

```diff
@@ -0,0 +1,102 @@
+// Created by matthew_cheok on 2/4/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import EpoxyCore
+import UIKit
+
+/// A stack of arbitrary bar views that can handle user selection.
+public final class SelectableStackView: BarStackView, StyledView {
+
+  // MARK: Lifecycle
+
+  /// - Parameters:
+  ///   - style: The style to be used.
+  required public init(style: Style) {
+    self.style = style
+    super.init(zOrder: .topToBottom)
+  }
+
+  // MARK: Public
+
+  public override func touchesBegan(_ touches: Set<UITouch>, with event: UIEvent?) {
+    super.touchesBegan(touches, with: event)
+    guard let location = touches.first?.location(in: self) else { return }
+
+    for wrapper in wrappers {
+      guard let converted = wrapper.view?.convert(location, from: self) else { continue }
+      if wrapper.view?.point(inside: converted, with: event) == true {
+        selectedWrapper = wrapper
+        break
+      }
+    }
+
+    updateHighlighting()
+  }
+
+  public override func touchesMoved(_ touches: Set<UITouch>, with event: UIEvent?) {
+    super.touchesMoved(touches, with: event)
+    guard
+      let wrapper = selectedWrapper,
+      let location = touches.first?.location(in: self),
+      let converted = wrapper.view?.convert(location, from: self)
+      else { return }
+
+    if wrapper.view?.point(inside: converted, with: event) == false {
+      selectedWrapper = nil
+      updateHighlighting()
+    }
+  }
+
+  public override func touchesEnded(_ touches: Set<UITouch>, with event: UIEvent?) {
+    super.touchesEnded(touches, with: event)
+
+    selectedWrapper?.handleSelection(animated: false)
+    selectedWrapper = nil
+    updateHighlighting()
+  }
+
+  // MARK: Private
+
+  private let style: Style
+  private var selectedWrapper: BarWrapperView?
+
+  private func updateHighlighting() {
+    for wrapper in wrappers {
+      wrapper.view?.backgroundColor = (selectedWrapper === wrapper)
+        ? style.selectedBackgroundColor
+        : .clear
+    }
+  }
+}
+
+// MARK: - StyledView
+
+extension SelectableStackView {
+  public struct Style: Hashable {
+    var selectedBackgroundColor: UIColor
+
+    public init(selectedBackgroundColor: UIColor) {
+      self.selectedBackgroundColor = selectedBackgroundColor
+    }
+  }
+}
+
+// MARK: - ContentConfigurableView
+
+extension SelectableStackView: ContentConfigurableView {
+  public struct Content: Equatable {
+    let models: [BarModeling]
```

Comment:
Since this is a `struct` with a public memberwise initializer let's make `models` a `public var`
```suggestion
    public var models: [BarModeling]
```